var/spack/repos/builtin/packages/jblob/package.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import fileinput, os, shutil
+
+class Jblob(Package):
+    """The German Climate Computing Center (DKRZ: Deutsches Klimarechenzentrum GmbH) 
+       dsaprovides a Long Term Archiving Service for large research data sets which are relevant for climate or Earth system research.
+    """
+
+    homepage = "https://cera-www.dkrz.de/WDCC/ui/cerasearch"
+    url      = "http://cera-www.dkrz.de/jblob/jblob-3.0.zip"
+
+    maintainers = ['ajkotobi']
+
+    version('3.0', sha256='576b5956358386a8832c6d1d13c410705e54888354a10cfd4f094513458067e4', expand=True)
+
+    depends_on('openjdk@1.8.0_202-b08:', type=('build', 'link', 'run'))
```

Comment:
Does this work with other Java providers? In that case, I would do:
```suggestion
    depends_on('java@8:', type='run')
```
Java is only needed at run-time since you aren't building anything.